db/scorers/ndcg@10.js

```diff
@@ -30,5 +30,9 @@ function nDCG(vals, ideal, k) {
   return d ? (n / d) : 0;
 }
 
-setScore(nDCG(scores, ideal, k));
+let ideal_scores = scores.slice();
```

Comment:
This will only count docs in `scores`, the top-N windows of results. nDCG requires knowing the global ideal DCG, the best docs available in the entire corpus, not just the top-N. I don't think this code will produce the expected behavior of nDCG.